utils/build/build.sh

```diff
@@ -188,6 +188,15 @@ build() {
             SYSTEM_TESTS_LIBDDWAF_VERSION=$(docker run --rm system_tests/weblog cat SYSTEM_TESTS_LIBDDWAF_VERSION)
             SYSTEM_TESTS_APPSEC_EVENT_RULES_VERSION=$(docker run --rm system_tests/weblog cat SYSTEM_TESTS_APPSEC_EVENT_RULES_VERSION)
 
+            # Special logic to turn off DD_DATA_STREAMS_ENABLED for spring-boot-native right now
+            # as having it turned on seems to break other existing tests due to a reflection issue
+            # in a library we use.
+            if [ "${WEBLOG_VARIANT}" = "spring-boot-native" ]  || [ "${WEBLOG_VARIANT}" = "spring-boot-3-native" ]; then
```

Comment:
if it's related to a given weblog, then define this env directly in the weblog dockerfile. Build.sh must remain generic

Reply:
removed